packages/e2e/cypress/integration/ui/components/authenticator/sign-up-with-phone/sign-up-with-phone.steps.ts

```diff
@@ -17,6 +17,7 @@ Then("I don't see {string} as an input field", (name: string) => {
 });
 
 When('I type the phone number {string}', (phone: string) => {
+  cy.findByRole('combobox', { name: /country code/i }).select('+1');
```

Comment:
This may belong in a separate `When I select the country code '+1'` command because we'll also test that `+1` is the default and that `8008675309` works without explicitly selecting the country code.

Reply:
Yeah that makes sense. Any thoughts on where the `'+1'` comes from in the feature file? Would we just hard code it?